src/Items.py

```diff
@@ -25,6 +25,13 @@
 
 # add sequential generated ids to the lists
 for key, val in enumerate(item_table):
+    if "id" in item_table[key]:
+        item_id = item_table[key]["id"]
+        if item_id > count:
```

Comment:
I don't think we should throw an error if `item_id == count` since that is just the default behavior.

Reply:
Also, is there any way to account for someone not putting the item IDs in order, but listing valid numbers? E.g. [1, 2, blank, 4, 3] will break, but could be solved by sorting the items to be [1, 2, 3, 4, blank].

Reply:
I'm not sure why someone would even want to have their custom ids out of order. That sounds like it should be something they should fix -- for their own sanity. 😜

Reply:
If we allow people to go backwards, it means we need to do collision detection on every single increment, and that's much more expensive than just asking people to have things in order